app/assets/stylesheets/groups.scss

```diff
@@ -274,10 +274,24 @@
   }
 }
 
-.group-invite-link-text {
+#collapseExample{
```

Comment:
can you pls give it a meaningful-name instead of `collapseExample`

Reply:
I just used the `div id` which was defined previously before I make changes. It is okay if I change that!?

Reply:
@tanmoysrt any inputs !?

Reply:
You can provide a better id for this

Reply:
Updated !